### DISCUSSION TITLE
Expected to add Binance Web3 Wallet

### DISCUSSION THREAD

**Discussion by LukeAntarctic** [rainbow-me/rainbowkit#2076] [Ideas]
https://www.binance.com/en/web3wallet

**Comment by 0xmkh** [comment]
We're supporting it right now. Check it out [here](https://www.rainbowkit.com/docs/custom-wallet-list#binance-web3-wallet).

**Reply by pipinstall1** [reply]
What version is it supported on? 2.1.3 seems to have that missing

**Reply by DanielSinclair** [reply]
This is released as v2.1.4
